articles/applications/application-settings/native/index.md

```diff
@@ -22,6 +22,8 @@ When creating an Auth0 Application, you'll be asked to indicate the *type* of Ap
 
 For desktop or mobile apps running natively on the device, you'll want to create a Native Application.
 
+Validate that the Allowed Web Origins URLs do not point to localhost.
```

Comment:
I'm not sure this should go here.... 🤔 
The page list every setting bellow, so if we are adding recommendations here, we should do for Allowed Web Origins and Allowed Origins (CORS) as well. Probably for logout url, too, although we don't have that in the production checks, not sure.
But also the recommendation only applies to production environments, since using localhost while in development is fine.

Reply:
Added into the settings section for each of the applicable URLs.